src/main/groovy/grails/plugins/cacheheaders/CacheHeadersTrait.groovy

```diff
@@ -7,10 +7,8 @@ import groovy.transform.*
 @CompileStatic
 trait CacheHeadersTrait extends ServletAttributes {
 
-	@Autowired
```

Comment:
It was.  Using the Autowired annotation causes issues running unit tests in projects dependent on this plugin.  There was no apparent loss of functionality upon removal (and there's another pull request to this repo that includes only this change for that same reason), so I considered it a reasonable assumption that it could be safely removed.